main/xvidcore/template.py

```diff
@@ -1,26 +1,21 @@
 pkgname = "xvidcore"
 pkgver = "1.3.7"
-pkgrel = 0
+pkgrel = 1
 build_wrksrc = "build/generic"
 build_style = "gnu_configure"
 make_cmd = "gmake"
 make_dir = "."
-hostmakedepends = ["gmake", "nasm"]
+hostmakedepends = ["automake", "gmake", "libtool", "nasm"]
 pkgdesc = "ISO MPEG-4 compliant video codec"
 maintainer = "q66 <q66@chimera-linux.org>"
 license = "GPL-2.0-or-later"
 url = "https://www.xvid.com"
 source = f"https://downloads.xvid.com/downloads/{pkgname}-{pkgver}.tar.bz2"
 sha256 = "aeeaae952d4db395249839a3bd03841d6844843f5a4f84c271ff88f7aa1acff7"
-# FIXME check
```

Comment:
it says fixme check but check is also disabled so what was the check

Reply:
idk i tested it and it's working fine, i assumed it just meant "check if it works fine"

Reply:
29243c5b42455b05ebea9f1987bae736feb9cf21, can't find anything specific